mappings/net/minecraft/util/dynamic/Codecs.mapping

```diff
@@ -26,10 +26,52 @@ CLASS net/minecraft/class_5699 net/minecraft/util/dynamic/Codecs
 		ARG 1 element
 	METHOD method_32852 (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Z
 		ARG 1 element
+	METHOD method_33817 xor (Lcom/mojang/serialization/Codec;Lcom/mojang/serialization/Codec;)Lcom/mojang/serialization/Codec;
+		COMMENT Returns a exclusive-or codec for {@link Either} instances.
+		COMMENT
+		COMMENT <p>This returned codec fails if both the {@code first} and {@code second} codecs can
+		COMMENT decode the input, while DFU's {@link com.mojang.serialization.codecs.EitherCodec}
+		COMMENT will always take the first decoded result when it is available.
+		COMMENT
+		COMMENT <p>Otherwise, this behaves the same as the either codec.
+		COMMENT
+		COMMENT @param <F> the first type
+		COMMENT @param <S> the second type
+		COMMENT @return the xor codec for the two codecs
+		COMMENT @see Codec#either(Codec, Codec)
+		COMMENT @see com.mojang.serialization.codecs.EitherCodec
+		ARG 0 first
+			COMMENT the first codec
+		ARG 1 second
+			COMMENT the second codec
 	CLASS 1
 		METHOD read (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Lcom/mojang/serialization/DataResult;
 			ARG 1 ops
 			ARG 2 input
 		METHOD write (Lcom/mojang/serialization/DynamicOps;Ljava/lang/Object;)Ljava/lang/Object;
 			ARG 1 ops
 			ARG 2 value
+	CLASS class_5839 XorCodec
```

Comment:
```suggestion
	CLASS class_5839 Xor
```

Codec suffix is redundant in a class named `Codecs`

Reply:
mojang tostring says `XorCodec`

Reply:
I think that's a pretty weak justification, also consider GameRules does not suffix the inner classes except for the rule subclasses I believe.